src/components/Profile.js

```diff
@@ -30,6 +31,12 @@ class Profile extends Component {
     });
   }
 
+  deleteRow(name) {
+    const index = this.state.uesr_items.findIndex((user_items) => {
```

Comment:
This will need to call `this.setState(...)` in order to set the state/delete the row.